packages/contracts/utils/etherscan.ts

```diff
@@ -1,6 +1,15 @@
 import networks from '../networks';
+import {
+  AllDeployments,
+  NameAndAddress,
+  CollectedProxyWithImplementation,
+} from '../types/etherscan';
+import {ChainConfig} from '@nomicfoundation/hardhat-verify/dist/src/types';
+import axios, {AxiosRequestConfig} from 'axios';
 import fs from 'fs';
 import HRE from 'hardhat';
+import {HardhatRuntimeEnvironment} from 'hardhat/types';
+import qs from 'qs';
```

Comment:
Shouldn't `qs` be included as a `devDependency`?

Reply:
Removed QS